src/components/sdk/ContentShare/index.tsx

```diff
@@ -10,37 +10,49 @@ import { BaseSdkProps } from '../Base';
 
 interface Props extends BaseSdkProps {
   nameplate?: string;
+  tileId?: number;
```

Comment:
Note that since the `VideoGrid` UI Component only supports one featured area/tile, when there's two content shares available, I split the featured area into two parts to display both content share tiles there. This change only exists in meeting demo  (I implemented a custom VideoTileGrid component in demo app) for testing purpose. 

For builders, they will need to create VideoTileGrid to handle multiple content share tiles based on their UX using updated `ContentTile` component.